custom/icds_reports/static/js/directives/month-filter/month-filter.directive.js

```diff
@@ -83,6 +77,10 @@ function MonthFilterController($scope, $location, $uibModal, storageService, dat
     var vm = this;
 
     // used by mobile dashboard
+    var isSDD =  $location.path().indexOf('service_delivery_dashboard') !== -1;
```

Comment:
(no need to do this, but wondered whether it also made sense to pull this logic to a helper...)

Reply:
Hope above comment resolved this question

Reply:
i'm not sure it does. you could still have `isSDD = navigationService.isSDD()` no? (anyways, no big deal)